lib/bootscale/file_storage.rb

```diff
@@ -13,11 +13,13 @@ def load(load_path)
 
     def dump(load_path, cache)
       path = cache_path(load_path)
-      return if File.exist?(path)
+      raise "This should not happen" if File.exist?(path)
```

Comment:
I though dump only gets called when load did not work, so that means the path did not exist ?